FeedSX/screens/createPost/index.tsx

```diff
@@ -0,0 +1,549 @@
+            }
+            style={{
+              opacity:
+                allAttachment?.length > 0 ||
+                formattedLinkAttachments?.length > 0 ||
+                postContentText != ''
+                  ? 1
+                  : 0.5,
+            }}
+            onPress={() => {
+              dispatch(
+                setUploadAttachments({
+                  mediaAttachmentData: allAttachment,
+                  linkAttachmentData: formattedLinkAttachments,
+                  postContentData: postContentText,
+                }) as any,
+              );
+              NavigationService.goBack();
+            }}>
+            <Text>POST</Text>
+          </TouchableOpacity>
+        }
+      />
+      <ScrollView>
+        {/* user profile section */}
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            paddingHorizontal: 15,
+            paddingTop: 12,
+          }}>
+          {/* profile image */}
+          <LMProfilePicture
+            fallbackText={memberData.name}
+            imageUrl={memberData.imageUrl}
+          />
+          {/* user name */}
+          <LMText
+            text={memberData.name}
+            textStyle={{
+              fontSize: 16,
+              fontWeight: '500',
+              color: '#222020',
+              marginLeft: 8,
+              textTransform: 'capitalize',
+            }}
+          />
+        </View>
+        {/* text input field */}
+        <LMInputText
+          placeholderText="Write something here..."
+          placeholderTextColor="#0F1E3D66"
+          inputTextStyle={{
+            marginHorizontal: 15,
+            marginVertical: 8,
+            fontSize: 16,
+            elevation: 0,
+          }}
+          multilineField
+          inputText={postContentText}
+          onType={val => {
+            setPostContentText(val);
+            detectLinks(postContentText);
+          }}
+        />
+        {/* selected media section */}
+        <View>
+          {/* multi media selection section */}
+          {formattedMediaAttachments ? (
+            formattedMediaAttachments?.length > 1 ? (
+              <LMCarousel
+                attachments={formattedMediaAttachments}
+                showCancel
+                onCancel={index => removeMediaAttachment(index)}
+              />
+            ) : (
+              <>
+                {/* single image selected section */}
+                {formattedMediaAttachments[0]?.attachmentMeta.format ===
+                  'image/jpg' && (
+                  <LMImage
+                    imageUrl={`${formattedMediaAttachments[0]?.attachmentMeta.url}`}
+                    showCancel
+                    onCancel={() => removeSingleAttachment()}
+                  />
+                )}
+                {/* single video selected section  */}
+                {formattedMediaAttachments[0]?.attachmentMeta.format ===
+                  'video/mp4' && (
```

Comment:
Better if you import from `string.ts` file

Reply:
resolved